### ISSUE TITLE
Imagenet label ids to (textual) classes

### ISSUE THREAD

**Issue by ngc92** [tensorflow/hub#133]
It is currently very difficult (or I have just looked in the wrong places) to find out the correspondence between the predicted labels of the imagenet modules, and the actual (textual) class of the image. 
The module documentation links to the [tf-slim modules](https://github.com/tensorflow/models/blob/master/research/slim/README.md#pre-trained-models), but this provides only the source code of the models. 
Is there a file somewhere that lists these correspondences? 
This should be linked in the module documentation, and ideally the translation from labels to classes should be programmatically accessibly from the module instance.

**Comment by andresusanopinto** [comment]
Hi @ngc92,

I did a quick search and it seems these labels are correct [https://raw.githubusercontent.com/kikirizki/tensorflow-nobazel/master/imagenet_slim_labels.txt].

Leaivng this open so module documentation is updated with a more formal and trustworthy source of labels.

**Comment by hgadig** [comment]
@andresusanopinto  Is this open for the module owner to update the documentation or arnoegw is the one to update it ? Could you please confirm ?

**Comment by arnoegw** [comment]
I'm on it.

**Comment by arnoegw** [comment]
Docs for image classification modules like https://tfhub.dev/google/imagenet/inception_v3/classification/1 have now been updated wityh a linkt to download.tensorflow.org/data/ImageNetLabels.txt.
